thirdparty/CMakeLists.txt

```diff
@@ -30,8 +30,16 @@ else()
 endif()
 message( STATUS "Thirdparty downloaded file path: ${THIRDPARTY_DOWNLOAD_PATH}" )
 # ----------------------------------------------------------------------
-# Find pthreads
+# assume built-in pthreads on MacOS
+IF(APPLE)
```

Comment:
please use same format style with others,
use "if ( xxx )" instead of "IF(xxx)";
use "endif ()" instead of "ENDIF()"

Reply:
Done.